### DISCUSSION TITLE
My vercel postgres is not available

### DISCUSSION THREAD

**Discussion by skymins04** [vercel/community#5891] [Help]
### Summary

The status of my postgres DB appears as "available", but it keeps time-out when I connect. My postgres dashboard shows an error toast with the words "Failed to fetch tables in your database".

### Steps to Reproduce

<img width="365" alt="스크린샷 2024-02-21 오후 9 02 47" src="https://github.com/vercel/community/assets/33021438/bf4a4c21-6955-419e-9737-847d08eb6685">

<img width="522" alt="스크린샷 2024-02-21 오후 8 59 31" src="https://github.com/vercel/community/assets/33021438/e2bb9675-b066-4448-8106-ad96c9a26e20">

**Comment by Martin2037** [comment]
timeout..... me too

**Reply by sieudang99plenti** [reply]
me too
